### DISCUSSION TITLE
frePPLe 8.6.0 doker issue

### DISCUSSION THREAD

**Discussion by Samee681** [frePPLe/frepple#649] [Q&A] [answered]
I have connection problem with database, if I run:

```
 sudo docker run \
  -e POSTGRES_HOST=127.0.0.1\
  -e POSTGRES_PORT=5432 \
  -e POSTGRES_USER=frepple\
  -e POSTGRES_PASSWORD=frepple\
  --name frepple-community-local \
  --publish 9091:80 \
  --detach \
  ghcr.io/frepple/frepple-community:latest
```
  
Frepple try to connect to 127.0.0.1 and after 8-9 seconds the container disapper. 
Here the log:
Waiting for the database to be ready.

I have tested on Linux (ubuntu 24.04) and Windows 11 as well.
I have the same issue on both system.
How could it be solved?

**Comment by jdetaeye** [comment, marked as answer]
The postgres host 127.0.0.1 is resolved as the docker container itself, and there is no postgres database in the frepple container.
Try using "host.docker.internal" as POSTGRES_HOST.    This will be resolved as the host machine that is running docker.

**Reply by Samee681** [reply]
Thank you very much, the quick answer.
